### PR TITLE
week 6 dev updates and project proposal

### PR DESCRIPTION
**Pull request by rupam-04**

### CODE REVIEW

**Comment by taxmeifyoucan** [comment]
Looks great, thanks for the project proposal

File: development-updates.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ With the gained insight into the protocol, the following weeks serve as a deep d
...
 | [Rose Jethani](https://github.com/rose2221)                  | [Update 3](https://hackmd.io/@0xrosetteeee/Week3)               | [Update 4](https://hackmd.io/@0xrosetteeee/Week4)               | [Update 5](https://hackmd.io/@0xrosetteeee/BJw067Eu0)           |                                                                                            |
-| [Rupam Dey](https://github.com/rupam-04)                     | [Update 3](https://hackmd.io/@rupam-04/Week3)                   | [Update 4](https://hackmd.io/@rupam-04/Week4)                   | [Update 5](https://hackmd.io/@rupam-04/Week5)                   |                                                                                            |
+| [Rupam Dey](https://github.com/rupam-04)                     | [Update 3](https://hackmd.io/@rupam-04/Week3)                   | [Update 4](https://hackmd.io/@rupam-04/Week4)                   | [Update 5](https://hackmd.io/@rupam-04/Week5)                   | [Light Client Support in Prysm](https://hackmd.io/@Bastin/BkdVcRQvA)                                                                                           |
```
Comment (taxmeifyoucan) [inline development-updates.md:115]: you should link the proposal within repo `projects/light-client-support-in-prysm.md`

Comment (rupam-04) [reply development-updates.md:115]: > you should link the proposal within repo `projects/light-client-support-in-prysm.md`

done!

**Comment by rupam-04** [comment]
@taxmeifyoucan done now

**Pull request merged**